core/templates/pages/admin-page/admin-page.module.ts

```diff
@@ -81,11 +81,11 @@ class AdminPageModule {
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 import { downgradeModule } from '@angular/upgrade/static';
 
-const bootstrapFn = (extraProviders: StaticProvider[]) => {
+const bootstrapFnAsync = async(extraProviders: StaticProvider[]) => {
```

Comment:
Ditto?

Reply:
Same, as it is an anonymous function, a space cannot be included after async.

Reply:
Thanks for the explanation! :)